frontend/src/metabase/components/MetadataInfo/TableInfo/TableInfo.unit.spec.tsx

```diff
@@ -69,19 +84,23 @@ describe("TableInfo", () => {
 
   it("should fetch fks if fks are undefined on table", () => {
     setup({
-      id: PRODUCTS.id,
-      table: new Table({ ...PRODUCTS, fields: [1, 2, 3], fks: undefined }),
+      id: PRODUCTS_ID,
+      table: new Table({
```

Comment:
Could you go through metadata creation and not create metabase-lib entities directly?